Gemfile.lock

```diff
@@ -242,96 +242,95 @@ GEM
       rest-client (>= 1.6.7)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
-    method_source (0.9.2)
-    mime-types (3.3)
+    method_source (1.0.0)
+    mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2019.1009)
-    mimemagic (0.3.3)
-    mini_magick (4.9.5)
+    mimemagic (0.3.4)
+    mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.13.0)
-    msgpack (1.3.1)
+    minitest (5.14.0)
+    msgpack (1.3.3)
     mysql2 (0.5.3)
     net-scp (2.0.0)
       net-ssh (>= 2.6.5, < 6.0.0)
     net-ssh (5.2.0)
     netrc (0.11.0)
     nio4r (2.5.2)
-    nokogiri (1.10.8)
+    nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
-    oj (3.10.0)
+    oj (3.10.5)
     orm_adapter (0.5.0)
     parallel (1.19.1)
     parser (2.6.5.0)
       ast (~> 2.4.0)
     passenger (6.0.4)
       rack
       rake (>= 0.8.1)
-    polyamorous (2.3.0)
-      activerecord (>= 5.0)
+    polyamorous (2.3.2)
+      activerecord (>= 5.2.1)
     psych (3.1.0)
-    public_suffix (4.0.1)
+    public_suffix (4.0.3)
     puma (4.3.3)
       nio4r (~> 2.0)
-    rack (2.0.8)
+    rack (2.2.2)
     rack-cors (0.4.1)
     rack-proxy (0.6.5)
       rack
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (6.0.2.1)
-      actioncable (= 6.0.2.1)
-      actionmailbox (= 6.0.2.1)
-      actionmailer (= 6.0.2.1)
-      actionpack (= 6.0.2.1)
-      actiontext (= 6.0.2.1)
-      actionview (= 6.0.2.1)
-      activejob (= 6.0.2.1)
-      activemodel (= 6.0.2.1)
-      activerecord (= 6.0.2.1)
-      activestorage (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    rails (6.0.2.2)
+      actioncable (= 6.0.2.2)
+      actionmailbox (= 6.0.2.2)
+      actionmailer (= 6.0.2.2)
+      actionpack (= 6.0.2.2)
+      actiontext (= 6.0.2.2)
+      actionview (= 6.0.2.2)
+      activejob (= 6.0.2.2)
+      activemodel (= 6.0.2.2)
+      activerecord (= 6.0.2.2)
+      activestorage (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       bundler (>= 1.3.0)
-      railties (= 6.0.2.1)
+      railties (= 6.0.2.2)
       sprockets-rails (>= 2.0.0)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    rails_best_practices (1.19.4)
+    rails_best_practices (1.19.5)
```

Comment:
### [rails_best_practices](http://rails-bestpractices.com)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.19.4 → 1.19.5

[_~~change-log~~, [source-code](http://github.com/railsbp/rails_best_practices)_]





<details>
<summary>Commits</summary>

A change of **14** commits. See the full changes on [the compare page](https://github.com/flyerhzm/rails_best_practices/compare/v1.19.4...v1.19.5).

These are the first 10 commits:
- (a264f1a) [Auto corrected by following Ruby EmptyLine](https://github.com/flyerhzm/rails_best_practices/commit/a264f1aa9c89708a2928d132c65dac8af2592f6a)
- (e0ae818) [Add bigint to AlwaysAddDbIndexReview](https://github.com/flyerhzm/rails_best_practices/commit/e0ae8186e55347e1532fe975002ad4f4aa85ddb1)
- (bdbd3ee) [Merge pull request #347 from r-uchida13/add_bigint_to_always…](https://github.com/flyerhzm/rails_best_practices/commit/bdbd3ee08a1b73993d1881e1407301b7f1c3352c)
- (bc05be5) [Merge pull request #343 from flyerhzm/awesomecode-ruby/empty…](https://github.com/flyerhzm/rails_best_practices/commit/bc05be5c203ae3203ef967c457cb3b137a1a5f2b)
- (2482a0f) [add the link format for vscode](https://github.com/flyerhzm/rails_best_practices/commit/2482a0fed52e931bb76579bc849161a7fc02f296)
- (90d8e2c) [change the doc & parser](https://github.com/flyerhzm/rails_best_practices/commit/90d8e2c299fa7e1f983b88721ab3fa048480d8df)
- (c122273) [fix the url format](https://github.com/flyerhzm/rails_best_practices/commit/c1222736b080191e23e498844c673d690f379a19)
- (71cdd5a) [upgrade the version number](https://github.com/flyerhzm/rails_best_practices/commit/71cdd5a5673dee2b19f4e1a8ef826ca7d3c68fb2)
- (39b95b8) [Merge pull request #349 from angezanetti/AddVscodeOption](https://github.com/flyerhzm/rails_best_practices/commit/39b95b819a2d0ed98ef5ef3bd4931679857c2e11)
- (8bcfb91) [update .travis.yml](https://github.com/flyerhzm/rails_best_practices/commit/8bcfb916f18689f77eb2559ed92641edc4f8bd95)


</details>